lemminx-maven/src/test/java/org/eclipse/lemminx/maven/test/RemoteRepositoryTest.java

```diff
@@ -117,6 +117,25 @@ public void testDownloadArtifactOnHover() throws IOException, InterruptedExcepti
  		assertTrue(hover.getContents().getRight().getValue().contains(description));
 	}
 	
+	@Test(timeout=15000)
+ 	public void testDownloadNonCentralArtifactOnHover() throws IOException, InterruptedException, ExecutionException, URISyntaxException {
+		languageService.initializeIfNeeded();
+		File mavenRepo = languageService.getExtensions().stream() //
+			.filter(MavenLemminxExtension.class::isInstance) //
+			.map(MavenLemminxExtension.class::cast) //
+			.findAny() //
+			.map(mavenLemminxPlugin -> mavenLemminxPlugin.getMavenSession().getRepositorySession().getLocalRepository().getBasedir())
+			.get();
+		File artifactDirectory = new File(mavenRepo, "jascut/jascut/0.0.3");
+		final DOMDocument document = createDOMDocument("/pom-remote-artifact-non-central-download-hover.xml", languageService);
+		final Position position = new Position(14, 18);
+		assertFalse(artifactDirectory.exists());
+ 		languageService.doHover(document, position, new SharedSettings());
```

Comment:
I would have liked to try and see if the description of the artifact was being retrieved, but it doesn't seem to have any description in the jascut's pom.xml (I downloaded it manually from soffid).

Maybe this is just an issue with certain jars such as jascut and svnkit, perhaps they aren't packaged correctly?